packages/appstore/src/components/app.tsx

```diff
@@ -1,45 +1,24 @@
-import classNames from 'classnames';
-import * as React from 'react';
-import { setWebsocket, routes } from '@deriv/shared';
-import { StoreProvider, observer } from '@deriv/stores';
+import React from 'react';
 import CashierStoreProvider from '@deriv/cashier/src/cashier-providers';
 import CFDStoreProvider from '@deriv/cfd/src/cfd-providers';
-import Routes from 'Components/routes/routes';
-import { useStores, initContext } from 'Stores';
-import { TRootStore } from 'Types';
+import { StoreProvider } from '@deriv/stores';
+import AppContent from './app-content';
 import './app.scss';
 
-type TAppProps = {
+type TProps = {
     passthrough: {
-        root_store: TRootStore;
-        WS: Record<string, any>;
+        root_store: React.ComponentProps<typeof StoreProvider>['store'];
     };
 };
 
-const App = ({ passthrough: { WS, root_store } }: TAppProps) => {
-    initContext(root_store, WS);
-    setWebsocket(WS);
-    const { ui }: TRootStore = useStores();
+const App: React.FC<TProps> = ({ passthrough: { root_store } }: TProps) => (
```

Comment:
@farzin-deriv 
```suggestion
const App: React.FC<TProps> = ({ passthrough: { root_store } }) => (
```
🤔 do we need type this component twice?